src/coreclr/src/jit/lsra.cpp

```diff
@@ -5596,8 +5596,14 @@ void LinearScan::allocateRegisters()
             {
                 allocate = false;
             }
-            else if (refType == RefTypeParamDef && varDsc->lvRefCntWtd() <= BB_UNITY_WEIGHT)
-            {
+            else if (refType == RefTypeParamDef &&
```

Comment:
Nit: would it be correct to write the condition as 
```
            else if ((refType == RefTypeParamDef) && (varDsc->lvRefCntWtd() <= BB_UNITY_WEIGHT) &&
                     (!currentRefPosition->lastUse || (currentInterval->physReg == REG_STK)))
```
so it takes 2 lines instead of 3?